doc/data/messages/t/too-many-statements/good.py

```diff
@@ -1 +1,22 @@
-# This is a placeholder for correct code for this message.
+import random
+
+
+def distribute_candies(children, candies_per_child):
```

Comment:
Perhaps its secondary to the overall point of the example, but is `children` a list? Would the code run ok if we were to call this function?

Reply:
Let's type it as a list of Child without defining Child ?

Reply:
Sure sounds good!

Reply:
```suggestion
def distribute_candies(children: list[Child], candies_per_child: int):
```